packages/fast-tooling-react/src/css-editor/border/border.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import { get } from "lodash-es";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import {
+    BorderStyleValue,
+    CSSBorderHandledProps,
+    CSSBorderProps,
+    CSSBorderState,
+    CSSBorderUnhandledProps,
+} from "./border.props";
+
+export default class CSSBorder extends Foundation<
+    CSSBorderHandledProps,
+    CSSBorderUnhandledProps,
+    CSSBorderState
+> {
+    public static displayName: string = "CSSBorder";
+
+    protected handledProps: HandledProps<CSSBorderHandledProps> = {
+        data: void 0,
+        onChange: void 0,
+        managedClasses: void 0,
+    };
+
+    constructor(props: CSSBorderProps) {
+        super(props);
+
+        this.state = {
+            borderColor: get(this.props.data, "borderColor", "#000"),
+            borderStyle: get(this.props.data, "borderStyle", BorderStyleValue.solid),
+            borderWidth: get(this.props.data, "borderWidth", "1px"),
```

Comment:
I would suggest that you save these as private variables, then where you set state for componentDidUpdate, use those to make sure if they are ever undefined they revert back to the defaults